guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc

```diff
@@ -8,12 +8,12 @@ Use the following procedure to create an activation key that you can use to subs
 . In the *Name* field, enter *CentOS* or the name of your new activation key.
 . In the *Description* field, enter a description for the activation key.
 . From the *Environment* list, select the *Library*.
-. From the *Content View* list, select *Centos Content View*.
+. From the *Content View* list, select *Centos content view*.
```

Comment:
Unable to verify this right now, can someone test this?

Reply:
I will revert this change so it does not hold up the PR.

Reply:
I will revert this change so it does not hold up the PR.